src/util/customTransformer.js

```diff
@@ -78,6 +79,26 @@ async function runUserTransform(
     }),
   );
 
+  await jail.set(
+    '_getGeoLocation',
```

Comment:
I would name the function `geolocation(ip)` so it's more crisp when customer invokes it like `await geolocation(ip)`. Wdyt ?

Reply:
agreed